packages/agent/src/invitation/handler.ts

```diff
@@ -122,33 +118,6 @@ export const receiveCredentialFromOpenId4VciOffer = async ({
         })
       }
 
-      if (didMethod) {
-        const didResult = await agent.dids.create<JwkDidCreateOptions | KeyDidCreateOptions>({
-          method: didMethod,
-          options: {
-            key,
-          },
-        })
-
-        if (didResult.didState.state !== 'finished') {
-          throw new Error('DID creation failed.')
-        }
-
-        let verificationMethodId: string
-        if (didMethod === 'jwk') {
-          const didJwk = DidJwk.fromDid(didResult.didState.did)
-          verificationMethodId = didJwk.verificationMethodId
-        } else {
-          const didKey = DidKey.fromDid(didResult.didState.did)
-          verificationMethodId = `${didKey.did}#${didKey.key.fingerprint}`
-        }
-
-        return {
-          didUrl: verificationMethodId,
-          method: 'did',
-        }
-      }
-
```

Comment:
why did you remove this? This is needed for the Paradym WAllet?

Reply:
I don't think there was a reason, was just a bit too focussed on Funke wallet. Will revert.